resources/js/components/FormField.vue

```diff
@@ -1,58 +1,154 @@
 <template>
-    <default-field :field="field">
+
+    <default-field :field="field" class="color-picker">
+
         <template slot="field">
-            <div class="inline-flex mb-2">
-                <div class="color-input rounded-l-lg border-r-0 h-100 border border-60 color-input-value" v-bind:style="{ backgroundColor: value, width: '36px' }"></div>
-                <input :id="field.name" type="text"
-                       class="w-25 form-control form-input form-input-bordered color-input rounded-l-none"
-                       :class="errorClasses"
-                       :placeholder="field.name"
-                       v-model="value"
-                />
+
+            <div ref="container">
+
+                <div class="inline-flex mb-2" @click="showPicker">
+
+                    <div class="color-input rounded-l-lg border-r-0 h-100 border border-60 color-input-value"
+                         :style="{ backgroundColor: value, width: '36px' }"/>
+
+                    <input ref="input"
+                           :id="field.name"
+                           type="text"
+                           class="w-25 form-control form-input form-input-bordered color-input rounded-l-none"
+                           :class="errorClasses"
+                           :placeholder="field.name"
+                           v-model="value"/>
+
+                </div>
+
+                <component ref="colorpicker"
+                           v-if="shouldShowPicker"
+                           :is="component"
+                           :id="field.name"
+                           :class="[ errorClasses, { absolute: field.autoHidePicker && field.pickerType !== 'slider' } ]"
+                           :palette="palette"
+                           v-model="value"
+                           @input="handleChange"/>
+
+                <p v-if="hasError" class="my-2 text-danger">
+                    {{ firstError }}
+                </p>
+
             </div>
```

Comment:
Same here - it's really hard for me to tell what actually changed because of all the code style formatting changes.  Can you revert the formatting changes and only propose the actual code updates for RGBA support, please?